pandas/core/frame.py

```diff
@@ -8692,6 +8692,10 @@ def update(
         dict.update : Similar method for dictionaries.
         DataFrame.merge : For column(s)-on-column(s) operations.
 
+        Notes
+        --------
```

Comment:
I think having the `-------` longer than the title used to make the CI crash. Not sure why the CI is happy now, but maybe worth making `Notes` and it the same length in case it fails again in the future.

Reply:
Now it has the same size.